src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -159,6 +167,55 @@ export default {
                 this.changeFeatureTitle({ feature: this.feature, title: value })
             },
         },
+        geometry: {
+            get() {
+                /*
+                Openlayers polygons coordinates are in a triple array
+                The first array is the "ring", the second is to hold the coordinates, which are in an array themselves
+                We don't have rings in this case, so we need to create an ol geometry
+                */
+
+                const geom = [this.feature.coordinates]
+
+                return new Polygon(geom)
+            },
+        },
+        isClosed: {
+            get() {
```

Comment:
no need to declare a getter function if it's only used to get the information/the data, a get/set approach is when you want to link the result of the computed to a HTML Element, and have a handler/a callback when the user modify this value

Here you can simply write
```
isClosed() {
    return ...
}
```